src/DynamoRevit/DynamoRevit.cs

```diff
@@ -1052,11 +1052,12 @@ public static string GetRevitContext(DynamoRevitCommandData commandData)
 
             try
             {
-                DynamoModel.IsCrashing = true;
-                RevitDynamoModel.OnRequestsCrashPrompt(
-                    RevitDynamoModel,
-                    new CrashPromptArgs(args.Exception.Message + "\n\n" + args.Exception.StackTrace));
-                RevitDynamoViewModel.Exit(false); // don't allow cancellation
+                if (!args.Handled)
+                {
+                    DynamoModel.IsCrashing = true;
+                    RevitDynamoModel.OnRequestsCrashPrompt(new CrashErrorReportArgs(args.Exception));
+                    RevitDynamoViewModel.Exit(false); // don't allow cancellation
+                }
```

Comment:
Maybe we should still mark the arg.handled as true in the finally block? The docs still say it prevents some internal work thing to be triggered

Reply:
Yes, it's still being marked true inside the finally block.

Reply:
@aparajit-pratap Thanks, so this PR will solve the remaining part of DYN-6066 for now?

Reply:
@sm6srw yes, it fulfills the remaining part of DYN-6066 but I'm still testing something more for my task DYN-6492 as mentioned during standup.